openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx

```diff
@@ -209,10 +189,129 @@ const GlossaryPageV1 = () => {
         });
       })
       .finally(() => {
+        setIsLoading(false);
         handleChildLoading(false);
         setLoadingKey((pre) => {
-          return pre.filter((item) => item !== key);
+          return pre.filter((item) => item !== fqn);
+        });
+      });
+  };
+
+  const fetchGlossaryTermAssets = (fqn: string, forceReset = false) => {
```

Comment:
Need comments for the method and arguments

Reply:
Done